lib/facter/virtual.rb

```diff
@@ -160,7 +160,7 @@
       next "xenhvm"     if lines.any? {|l| l =~ /XenSource/ }
       next "hyperv"     if lines.any? {|l| l =~ /Microsoft Corporation Hyper-V/ }
       next "gce"        if lines.any? {|l| l =~ /Class 8007: Google, Inc/ }
-      next "kvm"        if lines.any? {|l| l =~ /virtio/i }
+      next "kvm"        if lines.any? {|l| l =~ /[Vv]irtio/i }
```

Comment:
Hmm, the `/i` on the regex means case-insensitive, so this shouldn't be needed. Can you provide an input string that doesn't match with the current code?